package.json

```diff
@@ -36,6 +36,7 @@
     "liftoff": "3.1.0",
     "lodash": "^4.17.15",
     "mkdirp": "^0.5.1",
+    "oracledb": "^4.0.1",
```

Comment:
It would be preferable to avoid adding this, but instead move tests to integration test suite and skipping it for non-oracle dbs.

Reply:
Where in the integrations folder should I place the tests?

Reply:
I'd say that `https://github.com/tgriesser/knex/tree/master/test/integration/builder`, probably`additional`, since I assume this is not a select per se?